python/ray/dag/compiled_dag_node.py

```diff
@@ -1302,7 +1302,8 @@ def _get_or_compile(
                 if not has_at_least_one_channel_input:
                     raise ValueError(
                         "Compiled DAGs require each task to take a ray.dag.InputNode "
-                        "or at least one other DAGNode as an input"
+                        "or at least one other DAGNode as an input. Invalid task node:\n"
```

Comment:
can you add a concrete action item? Like Add argument to a function `X` or something like that

Reply:
done